wajibisha/static/css/wajibisha.css

```diff
@@ -100,30 +120,35 @@ body {
   height: 350px;
   background-color: #fff; }
 
+.tweet-card {
+  height: 200px;
+  margin: 2rem;
+  border-color: #B7E4FF;
+  padding: 0; }
 
-span.input-group-btn {
-  position: absolute;
-  right: 8px;
-  top: 5px;
-  bottom:2px;
+.team {
+  padding: 0;
+  margin: 3rem;
+  height: 230px; }
+
+.input-group-btn {
+  position: relative;
+  top: 6.5px;
   z-index: 999;
-}
-input.form-control-lg {
-  height:3.2rem;
-}
-
-.clipped-header select{
-  width:250px;
-  background-color:#4CAE4E;
-  border-color:#4CAE4E;
-  color: white;
-}
-.tweet-card {
-  height:180px;
-  width:23rem;
-  margin:1rem;
-  padding:0;
-  border:5px solid; 
-  border-color:#BFD9E8!important;
-}
+  margin-left: -190px; }
+
+.input-group > .form-control {
+  height: 3.6rem;
+  border-color: darkgrey;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2); }
+
+input.form-control.form-control-lg.rounded::placeholder {
+  color: #d4d4d4;
+  padding-left: 2rem;
+  font-size: 0.8rem;
+  top: 6px; }
+
+select.form-control.form-control-lg {
+  color: white; }
 
+/*# sourceMappingURL=../../../../../../../../static/css/wajibisha.css.map */
```

Comment:
Any reason we're not going for a SASS file instead like the HURUmap base? - https://github.com/CodeForAfrica/HURUmap/tree/master/hurumap/static/css